test/tests/multiapps/picard_postprocessor/steady_main.i

```diff
@@ -78,6 +78,7 @@
     input_files = steady_sub.i
     clone_parent_mesh = true
     execute_on = 'timestep_begin'
+    keep_full_output_history = true
```

Comment:
what are you trying to preserve by adding this?
leave a comment for future maintainers

Reply:
Add comments to all of the postprocessor input files!